### PR TITLE
reformat remaining namespaces

### PR DESCRIPTION
**Pull request by rasom**
status: ready

### CODE REVIEW

File: src/status_im/commands/handlers/jail.cljs [hunk 1]
```diff
@@ -79,31 +79,31 @@
 
 ;; TODO(alwx): rewrite
 (reg-handler :command-handler!
-  (after (print-error-message! "Error on command handling"))
-  (handlers/side-effect! command-handler!))
+             (after (print-error-message! "Error on command handling"))
+             (handlers/side-effect! command-handler!))
```
Comment (dmitryn) [inline src/status_im/commands/handlers/jail.cljs:83]: seems like too much indent

File: src/status_im/translations/es_ar.cljs [hunk 2]
```diff
@@ -206,11 +204,8 @@
    :enter-address                         "Ingresar dirección"
    :send-transaction                      "Enviar transacción"
    :delete-contact                        "Eliminar contacto"
-   :mute-notifications                    "Silenciar notificaciones"
-
-
-   :contact-s                             {:one   "contacto"
-                                           :other "contactos"}
+   :mute-notifications                    "Silenciar notificaciones" :contact-s                             {:one   "contacto"
+                                                                                                             :other "contactos"}
```
Comment (dmitryn) [inline src/status_im/translations/es_ar.cljs]: incorrect new line formatting

File: src/status_im/translations/es_ar.cljs [hunk 3]
```diff
@@ -228,9 +223,6 @@
    :no-statuses-found                     "No se encontraron estados"
    :browsing-open-in-web-browser          "Abrir en navegador web"
    :delete-group-prompt                   "Esto no afectará a los contactos"
-   :edit-profile                          "Editar perfil"
-
-
-   :empty-topic                           "Tema vacío"
+   :edit-profile                          "Editar perfil" :empty-topic                           "Tema vacío"
```
Comment (dmitryn) [inline src/status_im/translations/es_ar.cljs]: new line formatting

File: src/status_im/translations/fr_ch.cljs [hunk 4]
```diff
@@ -203,11 +201,8 @@
    :enter-address                         "Renseigner l'adresse"
    :send-transaction                      "Envoyer la transaction"
    :delete-contact                        "Supprimer le contact"
-   :mute-notifications                    "Désactiver le son des notifications"
-
-
-   :contact-s                             {:one   "contact"
-                                           :other "contacts"}
+   :mute-notifications                    "Désactiver le son des notifications" :contact-s                             {:one   "contact"
+                                                                                                                        :other "contacts"}
```
Comment (dmitryn) [inline src/status_im/translations/fr_ch.cljs]: new line formatting

File: src/status_im/translations/fr_ch.cljs [hunk 5]
```diff
@@ -226,9 +221,6 @@
    :no-statuses-found                     "Aucun statut trouvé"
    :browsing-open-in-web-browser          "Ouvrir dans le navigateur"
    :delete-group-prompt                   "Cela n'affectera pas les contacts"
-   :edit-profile                          "Modifier le profil"
-
-
-   :empty-topic                           "Sujet vide"
+   :edit-profile                          "Modifier le profil" :empty-topic                           "Sujet vide"
```
Comment (dmitryn) [inline src/status_im/translations/fr_ch.cljs]: new line formatting

File: src/status_im/translations/hi.cljs [hunk 6]
```diff
@@ -203,11 +201,8 @@
    :enter-address                         "पता प्रविष्ट करें"
    :send-transaction                      "लेनदेन भेजें"
    :delete-contact                        "संपर्क मिटाएं"
-   :mute-notifications                    "अधिसूचनाएं म्यूट करें"
-
-
-   :contact-s                             {:one   "संपर्क"
-                                           :other "संपर्क"}
+   :mute-notifications                    "अधिसूचनाएं म्यूट करें" :contact-s                             {:one   "संपर्क"
```
Comment (dmitryn) [inline src/status_im/translations/hi.cljs]: new line formatting

File: src/status_im/translations/hi.cljs [hunk 7]
```diff
@@ -226,9 +221,6 @@
    :no-statuses-found                     "कोई स्टैटस नहीं मिले"
    :browsing-open-in-web-browser          "वेब ब्राउज़र में खोलें"
    :delete-group-prompt                   "इससे संपर्क प्रभावित नहीं होंगे"
-   :edit-profile                          "प्रोफाइल का संपादन करें"
-
-
-   :empty-topic                           "रिक्त विषय"
+   :edit-profile                          "प्रोफाइल का संपादन करें" :empty-topic                           "रिक्त विषय"
```
Comment (dmitryn) [inline src/status_im/translations/hi.cljs]: new line formatting

File: src/status_im/translations/hu.cljs [hunk 8]
```diff
@@ -203,11 +201,8 @@
    :enter-address                         "Cím megadása"
    :send-transaction                      "Tranzakció küldése"
    :delete-contact                        "Névjegy törlése"
-   :mute-notifications                    "Értesítések elnémítása"
-
-
-   :contact-s                             {:one   "névjegy"
-                                           :other "névjegyek"}
+   :mute-notifications                    "Értesítések elnémítása" :contact-s                             {:one   "névjegy"
```
Comment (dmitryn) [inline src/status_im/translations/hu.cljs]: new line formatting

File: src/status_im/translations/hu.cljs [hunk 9]
```diff
@@ -226,9 +221,6 @@
    :no-statuses-found                     "Nem található állapot"
    :browsing-open-in-web-browser          "Megnyitás új böngészőben"
    :delete-group-prompt                   "Nem lesz hatással a névjegyekre"
-   :edit-profile                          "Profil szerkesztése"
-
-
-   :empty-topic                           "Üres téma"
+   :edit-profile                          "Profil szerkesztése" :empty-topic                           "Üres téma"
```
Comment (dmitryn) [inline src/status_im/translations/hu.cljs]: new line formatting

File: src/status_im/translations/it_ch.cljs [hunk 10]
```diff
@@ -203,11 +201,8 @@
    :enter-address                         "Inserisci indirizzo"
    :send-transaction                      "Invia transazione"
    :delete-contact                        "Elimina contatto"
-   :mute-notifications                    "Silenzia le notifiche"
-
-
-   :contact-s                             {:one   "contatto"
-                                           :other "contatti"}
+   :mute-notifications                    "Silenzia le notifiche" :contact-s                             {:one   "contatto"
```
Comment (dmitryn) [inline src/status_im/translations/it_ch.cljs]: new line formatting

File: src/status_im/translations/it_ch.cljs [hunk 11]
```diff
@@ -226,9 +221,6 @@
    :no-statuses-found                     "Nessuno stato trovato"
    :browsing-open-in-web-browser          "Apri nel browser"
    :delete-group-prompt                   "Questo non intaccherà i contatti"
-   :edit-profile                          "Modifica profilo"
-
-
-   :empty-topic                           "Argomento non definito"
+   :edit-profile                          "Modifica profilo" :empty-topic                           "Argomento non definito"
```
Comment (dmitryn) [inline src/status_im/translations/it_ch.cljs]: new line formatting

File: src/status_im/translations/pl.cljs [hunk 12]
```diff
@@ -203,11 +201,8 @@
    :enter-address                         "Wprowadź adres"
    :send-transaction                      "Wyślij transakcję"
    :delete-contact                        "Usuń kontakt"
-   :mute-notifications                    "Wycisz powiadomienia"
-
-
-   :contact-s                             {:one   "kontakt"
-                                           :other "kontakty"}
+   :mute-notifications                    "Wycisz powiadomienia" :contact-s                             {:one   "kontakt"
```
Comment (dmitryn) [inline src/status_im/translations/pl.cljs]: new line formatting

File: src/status_im/translations/pl.cljs [hunk 13]
```diff
@@ -226,9 +221,6 @@
    :no-statuses-found                     "Nie znaleziono statusów"
    :browsing-open-in-web-browser          "Otwórz w przeglądarce"
    :delete-group-prompt                   "To nie będzie miało wpływu na kontakty"
-   :edit-profile                          "Edytuj profil"
-
-
-   :empty-topic                           "Pusty temat"
+   :edit-profile                          "Edytuj profil" :empty-topic                           "Pusty temat"
```
Comment (dmitryn) [inline src/status_im/translations/pl.cljs]: new line formatting

File: src/status_im/translations/pt_br.cljs [hunk 14]
```diff
@@ -203,11 +201,8 @@
    :enter-address                         "Inserir endereço"
    :send-transaction                      "Enviar transação"
    :delete-contact                        "Excluir contato"
-   :mute-notifications                    "Silenciar notificações"
-
-
-   :contact-s                             {:one   "contato"
-                                           :other "contatos"}
+   :mute-notifications                    "Silenciar notificações" :contact-s                             {:one   "contato"
+                                                                                                           :other "contatos"}
```
Comment (dmitryn) [inline src/status_im/translations/pt_br.cljs]: new line formatting

File: src/status_im/translations/pt_br.cljs [hunk 15]
```diff
@@ -226,9 +221,6 @@
    :no-statuses-found                     "Nenhum status encontrado"
    :browsing-open-in-web-browser          "Abrir no navegador de internet"
    :delete-group-prompt                   "Isso não afetará os contatos"
-   :edit-profile                          "Editar perfil"
-
-
-   :empty-topic                           "Assunto vazio"
+   :edit-profile                          "Editar perfil" :empty-topic                           "Assunto vazio"
```
Comment (dmitryn) [inline src/status_im/translations/pt_br.cljs]: new line formatting

File: src/status_im/translations/pt_pt.cljs [hunk 16]
```diff
@@ -203,11 +201,8 @@
    :enter-address                         "Digitar o endereço"
    :send-transaction                      "Enviar a transação"
    :delete-contact                        "Apagar contacto"
-   :mute-notifications                    "Silenciar as notificações"
-
-
-   :contact-s                             {:one   "contacto"
-                                           :other "Contactos"}
+   :mute-notifications                    "Silenciar as notificações" :contact-s                             {:one   "contacto"
+                                                                                                              :other "Contactos"}
```
Comment (dmitryn) [inline src/status_im/translations/pt_pt.cljs]: new line formatting

File: src/status_im/translations/sl.cljs [hunk 17]
```diff
@@ -226,9 +221,6 @@
    :no-statuses-found                     "Ni statusov"
    :browsing-open-in-web-browser          "Odpri v spletnem brskalniku"
    :delete-group-prompt                   "To ne bo vplivalo na stike"
-   :edit-profile                          "Uredi profil"
-
-
-   :empty-topic                           "Prazna tema"
+   :edit-profile                          "Uredi profil" :empty-topic                           "Prazna tema"
```
Comment (dmitryn) [inline src/status_im/translations/sl.cljs]: new line formatting

File: src/status_im/translations/sl.cljs [hunk 18]
```diff
@@ -203,11 +201,8 @@
    :enter-address                         "Vnesi naslov"
    :send-transaction                      "Pošlji transakcijo"
    :delete-contact                        "Izbriši stik"
-   :mute-notifications                    "Utišaj obvestila"
-
-
-   :contact-s                             {:one   "stik"
-                                           :other "stiki"}
+   :mute-notifications                    "Utišaj obvestila" :contact-s                             {:one   "stik"
+                                                                                                     :other "stiki"}
```
Comment (dmitryn) [inline src/status_im/translations/sl.cljs]: new line formatting

File: src/status_im/translations/ro.cljs [hunk 19]
```diff
@@ -226,9 +221,6 @@
    :no-statuses-found                     "Niciun status găsit"
    :browsing-open-in-web-browser          "Deschide în browser web"
    :delete-group-prompt                   "Aceasta nu va afecta contactele"
-   :edit-profile                          "Editare profil"
-
-
-   :empty-topic                           "Subiect gol"
+   :edit-profile                          "Editare profil" :empty-topic                           "Subiect gol"
```
Comment (dmitryn) [inline src/status_im/translations/ro.cljs]: new line formatting

File: src/status_im/translations/ro.cljs [hunk 20]
```diff
@@ -203,11 +201,8 @@
    :enter-address                         "Introduceți adresa"
    :send-transaction                      "Trimitere tranzacție"
    :delete-contact                        "Ștergere contact"
-   :mute-notifications                    "Notificări silențioase"
-
-
-   :contact-s                             {:one   "contact"
-                                           :other "contacte"}
+   :mute-notifications                    "Notificări silențioase" :contact-s                             {:one   "contact"
+                                                                                                           :other "contacte"}
```
Comment (dmitryn) [inline src/status_im/translations/ro.cljs]: new line formatting

File: src/status_im/translations/pt_pt.cljs [hunk 21]
```diff
@@ -226,9 +221,6 @@
    :no-statuses-found                     "Nenhum estado encontrado"
    :browsing-open-in-web-browser          "Abrir no navegador Web"
    :delete-group-prompt                   "Isto não afetará os contactos"
-   :edit-profile                          "Editar perfil"
-
-
-   :empty-topic                           "Tema em branco"
+   :edit-profile                          "Editar perfil" :empty-topic                           "Tema em branco"
```
Comment (dmitryn) [inline src/status_im/translations/pt_pt.cljs]: new line formatting

**Review by dmitryn — APPROVED** [review]

File: src/status_im/commands/handlers/jail.cljs (hunk 1 above)
Comment (rasom) [reply src/status_im/commands/handlers/jail.cljs:83]: that's actually correct indent https://github.com/bbatsov/clojure-style-guide#vertically-align-fn-args

Comment (dmitryn) [reply src/status_im/commands/handlers/jail.cljs:83]: @rasom correct, i was confused comparing with other `reg-handler` calls where args placed on second line

**Comment by rasom** [comment]
@dmitryn should be fixed now

**Review by flexsurfer — APPROVED** [review]

**Review by yenda — APPROVED** [review]

**Comment by rasom** [comment]
oh jeez, i forgot to squash commits... eh, anyway

**Pull request merged**